source/OptimaJet.DWKit.StarterApplication/Services/SendNotificationService.cs

```diff
@@ -0,0 +1,107 @@
+            NotificationRepository = notificationRepository;
+
+        }
+        public async Task SendNotificationToOrgAdminsAsync(Organization organization, String message, object subs)
+        {
+            var orgAdmins = UserRolesRepository.Get()
+                .Include(ur => ur.User)
+                .Include(ur => ur.Role)
+                .Where(ur => ur.OrganizationId == organization.Id && ur.RoleName == RoleName.OrganizationAdmin)
+                .ToList();
+            foreach (UserRole orgAdmin in orgAdmins)
+            {
+                await SendNotificationToUserAsync(orgAdmin.User, message, subs);
+            }
+        }
+        public async Task SendNotificationToUserAsync(User user, String message, object subs)
+        {
+            var notification = new Notification
+            {
+                MessageId = message,
+                UserId = user.Id,
+                MessageSubstitutions = subs
+            };
+            var updatedNotification = await NotificationRepository.CreateAsync(notification);
+            await HubContext.Clients.User(user.ExternalId).SendAsync("Notification", updatedNotification.Id);
+        }
+        public void NotificationEmailMonitor()
+        {
+            // Get limits from environment
+            int sendNotificationEmailMinutes = GetIntVarOrDefault("NOTIFICATION_SEND_EMAIL_MIN_MINUTES", 60);
+            int dontSendNotificationEmailMinutes = GetIntVarOrDefault("NOTIFICATION_SEND_EMAIL_MAX_MINUTES", 180);
+            var now = DateTime.UtcNow;
+            var notifications = NotificationRepository.Get()
+                                                    .Where(n => n.DateEmailSent == null
+                                                           && n.DateRead == null
+                                                           && n.User.EmailNotification == true
+                                                           && now.Subtract((DateTime)n.DateCreated).TotalMinutes > sendNotificationEmailMinutes
+                                                           && now.Subtract((DateTime)n.DateCreated).TotalMinutes < dontSendNotificationEmailMinutes
+                                                          )
+                                                    .Include(n => n.User)
+                                                    .ToList();
+            foreach (var notification in notifications)
+            {
+                SendEmailAsync(notification).Wait();
+            }
+        }
+        protected async Task SendEmailAsync(Notification notification)
+        {
+            var locale = CultureInfo.CurrentCulture.Name;
+            if (notification.User.Locale != null)
+            {
+                locale = notification.User.Locale;
+            }
+            locale = locale.ValidateLocale("en-US");
+            var subject = await Translator.TranslateAsync(locale, "notifications", "notifications.subject", null);
```

Comment:
I think I was going to comment on this, but not sure if I did.  Slack me about this one when you have the time.  I want to talk through this.

So what does `CultureInfo.CurrentCulture.Name` return?  The language of the server (e.g. setting LANG env var)?

Should this be a utility function or Extension method?  Not sure on what class.  Seems like `User.LocaleOrDefault` would be good.